usr/share/rear/layout/save/default/450_check_bootloader_files.sh

```diff
@@ -27,6 +27,9 @@ case $used_bootloader in
         #  - GRUB2 : SLES >= 12, RHEL >= 7, Ubuntu and other new Linux on POWER distro.
         CHECK_CONFIG_FILES=( ${CHECK_CONFIG_FILES[@]} /etc/lilo.conf /etc/yaboot.conf /etc/grub.cfg /etc/grub2.cfg /boot/grub2/grub2.cfg /boot/grub/grub.cfg)
         ;;
+    (ARM|ARM-ALLWINNER)
+        CHECK_CONFIG_FILES=( ${CHECK_CONFIG_FILES[@]} /boot/boot.scr )
+        ;;
     (*)
         BugError "Unknown bootloader ($used_bootloader) - ask for sponsoring to get this fixed"
```

Comment:
@schlomo 
See here.

Reply:
I see indeed. Good point. Let's leave it as you wrote it and maybe someday we will clean this up.